react/features/welcome/components/WelcomePage.web.js

```diff
@@ -144,6 +146,16 @@ class WelcomePage extends AbstractWelcomePage {
                                 { t('welcomepage.go') }
                             </Button>
                         </div>
+                        <div
```

Comment:
Why this div needs to be outside of the recent list component?

Reply:
the scss class navigate-section-list needed to be active on a div containing recent list because the property `position: relative` needed to be applied. However after reviewing this I realized this could be accomplished by applying that property in the container in base/react/components/SectionList.js

- [ ] remove extraneous divs and add the position relative property to SectionList highest level container